_posts/2020-12-07-r2dii-analysis-0-1-2-is-now-on-cran/r2dii-analysis-0-1-2-is-now-on-cran.Rmd

```diff
@@ -0,0 +1,172 @@
+---
```

Comment:
It'll be nice to have a `preview:` field pointing to an (CCO) image -- maybe something about bugs and holidays?